xml/Microsoft.VisualBasic/FileSystem.xml

```diff
@@ -400,6 +400,7 @@
         <Parameter Name="Attributes" Type="Microsoft.VisualBasic.FileAttribute" Index="1" />
       </Parameters>
       <Docs>
+        <param name="PathName">Optional. A string expression that specifies a file name, directory or folder name, or drive volume label. A zero-length string (<see langword="&quot;&quot;" />) is returned if <paramref name="PathName" /> is not found.</param>
```

Comment:
The param names are not equal so this should also be kept @joelmartinez

Reply:
I wasn't able to see what was going on with this, because the commit changed after the force push

![image](https://user-images.githubusercontent.com/90380/51916514-758c0080-23ab-11e9-8a4d-3f1f942d9c36.png)

And the github UI doesn't give a lot of context:

![image](https://user-images.githubusercontent.com/90380/51916548-88063a00-23ab-11e9-966f-109dfa6a8d55.png)